source/slang/slang-ir.cpp

```diff
@@ -3100,14 +3028,6 @@ IRDifferentialPairUserCodeType* IRBuilder::getDifferentialPairUserCodeType(
         operands);
 }
 
-IRBackwardDiffIntermediateContextType* IRBuilder::getBackwardDiffIntermediateContextType(
-    IRInst* func)
-{
-    if (!func)
-        func = getVoidValue();
```

Comment:
I think we shouldn't generate this function yet, as it's doing something slightly nonstandard

Reply:
Ack

Reply:
Should be fixed in the latest PR